lib/job/arBaseJob.class.php

```diff
@@ -135,7 +135,7 @@ private function formatLogMsg($message)
   /**
    * Clear various Qubit classes' caches.
    */
-  private function clearCache()
+  static function clearCache()
```

Comment:
I think this method makes more sense at this point put into lib/helper/QubitHelper.php as something named clear_all_orm_cache() or similar. It seems kind of weird to be in arBaseJob if it's being used way elsewhere in the code base.